ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml

```diff
@@ -124,6 +124,28 @@ images:
       - kas-artifacts
       - registry.ci.openshift.org/ocp/4.17:installer-kube-apiserver-artifacts
   to: installer
+- dockerfile_path: images/installer/Dockerfile.upi.ci
```

Comment:
vsphere ipi ci will never work without the upi-installer image, unless we go and replace it in all the steps that require it.